sparsity/sparse_frame.py

```diff
@@ -156,6 +157,17 @@ def copy(self, *args, **kwargs):
         return SparseFrame(self.data.copy(*args, **kwargs),
                            self.index.copy(*args, **kwargs),
                            self.columns.copy(*args, **kwargs))
+
+    def multiply(self, other):
```

Comment:
I know I suggested this but as we are under less time pressure now I thought about it and I think best would be to mimic pandas method: http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.multiply.html. So we will have easier time implementing dask.SparseFrame functionality. But let's simply use this for now.